Marlin/src/config/examples/Micromake/C1/basic/Configuration.h

```diff
@@ -1051,12 +1056,12 @@
 //
 // G20/G21 Inch mode support
 //
-//#define INCH_MODE_SUPPORT
+#define INCH_MODE_SUPPORT
 
 //
 // M149 Set temperature units support
 //
-//#define TEMPERATURE_UNITS_SUPPORT
+#define TEMPERATURE_UNITS_SUPPORT
```

Comment:
@MetalSearch — Is this in the factory config? Cool if it is. It's just an obscure feature, as is Inch Mode.